src/app/ReadClient.cpp

```diff
@@ -733,10 +751,12 @@ CHIP_ERROR ReadClient::ReadICDOperatingModeFromAttributeDataIB(TLV::TLVReader &&
     switch (operatingMode)
     {
     case Clusters::IcdManagement::OperatingModeEnum::kSit:
-        aType = PeerType::kNormal;
+        aMode = PeerOperatingMode::kNormal;
         break;
     case Clusters::IcdManagement::OperatingModeEnum::kLit:
-        aType = PeerType::kLITICD;
+        aMode = PeerOperatingMode::kLITICD;
+        // If operating mode is LIT, this device can send check-in message.
+        mPeerICD = true;
```

Comment:
Shouldn't we just call the member mPeerCanCheckIn, then?

Reply:
this change is removed, and unnecessary.